01-numpy.md

```diff
@@ -557,12 +568,12 @@ axes3). Once a subplot is created, the axes are can be titled using the
 Here are our three plots side by side:
 
 ~~~ {.python}
-import numpy
-import matplotlib.pyplot
+import numpy as np
```

Comment:
We have previously decided against this syntax. See the [FAI](https://github.com/swcarpentry/python-novice-inflammation/blob/gh-pages/instructors.md#frequently-argued-issues-fai) and PR #61.